pod/perlipc.pod

```diff
@@ -1132,9 +1132,9 @@ even if we aren't running setuid or setgid.  This is always a good idea
 for servers or any program run on behalf of someone else (like CGI
 scripts), because it lessens the chances that people from the outside will
 be able to compromise your system.
-Note that perl can be built without taint support,
-in which case -T silently does nothing
-(see L<perlsec> for how to check if your perl support taint checking).
+Note that perl can be built without taint support.  THere are two
```

Comment:
"THere"

Reply:
Shouldn't the preceding sentence be removed entirely?

Reply:
Hrm, that was in Neil's suggested text. I think it's still technically true, it's just you have go to the long way round with `-Accflags=...` rather than using a configure question.